x-pack/plugins/osquery/cypress/e2e/all/alerts.cy.ts

```diff
@@ -315,4 +301,42 @@ describe('Alert Event Details', () => {
       });
     });
   });
+
+  it('should substitute parameters in live query and increase number of ran queries', () => {
+    let initialNotificationCount: number;
+    let updatedNotificationCount: number;
+    loadAlertsEvents();
+    cy.getBySel('expand-event').first().click({ force: true });
+    cy.getBySel('osquery-actions-notification')
+      .should('not.have.text', '0')
+      .then((element) => {
+        initialNotificationCount = parseInt(element.text(), 10);
+      });
+    takeOsqueryActionWithParams();
+    cy.getBySel('osquery-empty-button').click();
+    cy.getBySel('osquery-actions-notification')
+      .should('not.have.text', '0')
+      .then((element) => {
+        updatedNotificationCount = parseInt(element.text(), 10);
+        expect(initialNotificationCount).to.be.equal(updatedNotificationCount - 1);
+      })
+      .then(() => {
+        cy.contains('Osquery Results').click();
+        cy.getBySel('osquery-results')
+          .within(() => {
+            cy.contains('tags');
+          })
+          .find(`[data-test-subj="osquery-results-comment"]`)
```

Comment:
why not cy.getBySel? is this any different?

Reply:
Good catch, unchained in favour of running the check inside `within`